example/index.html

```diff
@@ -38,6 +38,10 @@ <h1>Sankey with circular links</h1>
   <div id="chart"></div>
 
   <script>
+    // attach functions from d3-sankey-circular to main d3 object 
+    d3.sankeyCircular = sankeyCircular.sankeyCircular
+    d3.sankeyJustify = sankeyCircular.sankeyJustify
```

Comment:
there are two sub-optimal things about the config I have now. a user of `d3-sankey-circular` has to:
1) manually add any exports that she'd like to use to the main `d3` object.
2) access the exported function under the `sankeyCircular` name space, giving you this awkward, redundant `sankeyCircular.sankeyCircular`

I've fixed both of these problems before, but don't immediately remember how 😅

Reply:
@tomshanley I'd like to do some research / listen to other contributors and get these two things improved before merging this PR